zerver/tests/test_import_export.py

```diff
@@ -535,6 +536,15 @@ def test_import_realm(self) -> None:
             for realm_custom_profile_field in custom_profile_field]
         self.assertEqual(custom_profile_field_name[0], custom_profile_field_name[1])
 
+        # test realmauditlog
+        realmauditlogs = [
+            RealmAuditLog.objects.filter(realm=realm)
```

Comment:
Since we're effectively doing a single query for each realm here, it makes me think we should just have a loop that goes through each realm:

~~~
for realm in realms:
   logs = RealmAuditLog.objects.filter(realm=realm)
   self.assertEqual(...)
~~~

It's also not completely clear that we're populating this table in the test database.  It might be good to at least explicitly write one record to the table in the test.

Reply:
@showell in the `realms` [list](https://github.com/rheaparekh/zulip/blob/e92fada2e3c67057a8a4eed75eb587465d46054a/zerver/tests/test_import_export.py#L483), we only have two realms, the original realm and the realm we get after exporting and importing back the original realm. 

Here we are just having the sanity checks, like whether the imported realm contains the same stuff as the original realm, so we need comparison of the factors of both the realms and hence I am not looping through each realm.

Reply:
Yeah, I get that there are only two realms, but I still think the code will be cleaner with a loop.  Basically, I find it hard to understand the current code, which is essentially doing two parallel loops (inside the comprehensions), which seems needlessly complex to me.

Reply:
Okay, this is being followed throughout the test, so I'll add a new commit/PR and refactor the tests.